compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/window/Dialog.skiko.kt

```diff
@@ -46,12 +55,50 @@ import androidx.compose.ui.semantics.semantics
 import androidx.compose.ui.unit.IntRect
 import androidx.compose.ui.unit.IntSize
 import androidx.compose.ui.unit.center
+import kotlin.time.Duration.Companion.seconds
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.launch
 
 /**
  * The default scrim opacity.
  */
-private const val DefaultScrimOpacity = 0.6f
+private const val DefaultScrimOpacity = 0.78f
```

Comment:
Please revert default value here - it's picked from Android OS sources. It should be the same by default across the platforms